src/actions/categoryActions.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import {
+  CATEGORY_REQUEST,
+  CATEGORY_SUCCESS,
+  CATEGORY_FAIL,
+} from '../constants/categoryConstants';
+
+export const getCategories = async (dispatch) => {
+  dispatch({ type: CATEGORY_REQUEST });
+  try {
+    const res = await axios.get('/api/category');
+    dispatch({ type: CATEGORY_SUCCESS, payload: res.data.categories });
+  } catch (e) {
```

Comment:
You can make this as `catch(error)` to be consistent throughout the code. As you have used `catch(error)` everywhere.